coalib/output/ConsoleInteraction.py

```diff
@@ -316,7 +316,7 @@ def print_affected_files(console_printer,
                          file_dict,
                          color=True):
```

Comment:
Body: incorrect tense: should be Corrects, present tense. and it should be "a few", not "few".

also, the docstring add does not fit into this commit, so both remove it from the commit and from the commit body.